docs/src/auto_examples/tutorials/run_lda.py

```diff
@@ -278,7 +278,7 @@ def extract_documents(url='https://cs.nyu.edu/~roweis/data/nips12raw_str602.tgz'
 # methods on the blog at http://rare-technologies.com/lda-training-tips/ !
 #
 
-top_topics = model.top_topics(corpus)
+top_topics = model.top_topics(corpus) #, num_words=20)
```

Comment:
Also undesirable. We should probably revert whatever commit brought in those documentation changes.

Reply:
OK, will revert.

I think the cache on develop may be stale.